src/js/player.js

```diff
@@ -568,6 +568,13 @@ class Player extends Component {
       el.appendChild(tag);
 
       playerElIngest = this.playerElIngest_ = el;
+      // move properties over from our custom `video-js` element
+      // to our new `video` element. This will move things like
+      // `src` or `controls` that were set via js before the player
+      // was initialized.
+      Object.keys(el).forEach((k) => {
```

Comment:
actually, I wonder how much of this we want to do. I can see some benefit in having it but I can also see moving stuff like autoplay and src causing issues and having us enter into the `lateInit` part of the html5 tech without wanting to.
We should already be setting the source properly on the tech, are we not?

Reply:
So this fix basically makes things like this work:
```js
var vid = document.querySelector('video-js');

// this will only set the property on the element and not the attribute.
vid.src = 'some-src.mp4';
vid.autoplay = true;

videojs(vid);
```

Currently we only bring over attributes from the `<video-js>` element. This means that the above code will have no `src` and not `autoplay`. This is because we only move over attributes and the `<video-js>` element does not set attributes when properties are set like a real video element does.